### ISSUE TITLE
Update Readme.md

### ISSUE THREAD

**Issue by akashsara** [akashsara/homebrew_helper#15]
The readme is kinda useless as it is now. Update the readme to describe what the bot is & maybe some screenshots. Feel free to post here for any questions.

**Comment by ashutosh007865** [comment]
I am going to improve your ReadMe file. I definitely want to ask for your permission to assign this work to me but I am out of time due to the last day for hactoberfest. so I hope I can help you and as well as myself

**Comment by akashsara** [comment]
I've assigned the issue to you. Hope you manage to submit the PR before the deadline!

**Comment by ashutosh007865** [comment]
Thank you so much. Due to some timing issues, I made some basic changes. I
have lots of things to add to this project and I will do in the coming
days. but right now I just need one PR to be accepted.

**Comment by akashsara** [comment]
I've marked it as hacktoberfest-accepted. It should count towards your 4 PRs now. But it does need quite a few changes, please check the review of the PR.

**Comment by ashutosh007865** [comment]
yeah, I knew it. and I will try to contribute more and more when I get
some time.

**Comment by ashutosh007865** [comment]
And thank you so much for your favour

**Comment by ashutosh007865** [comment]
I am facing one while I am trying to run this project. I fully each and
every required as mentioned in the file but still, it's showing an error.
That's why I can't able to elaborate more in the readme file. Please help me

**Comment by ashutosh007865** [comment]
bro, I am facing one error while I am trying to run this project. I also
full fill all the requirements as mentioned in the text file. so please
help me out

**Comment by akashsara** [comment]
What's the error you're facing?
